src/UI/SlotArea.cpp

```diff
@@ -1932,6 +1940,16 @@ void cSlotAreaFurnace::OnSlotChanged(cItemGrid * a_ItemGrid, int a_SlotNum)
 
 void cSlotAreaFurnace::HandleSmeltItem(const cItem & a_Result, cPlayer & a_Player)
 {
+	if (& a_Player == m_Furnace->GetLastSmelter())
+	{
+		int Reward = m_Furnace->GetReward();
+		LOG("Reward %d\n", Reward);
```

Comment:
This should be a LOGD, if it's needed at all, we don't need to log an event every time a player takes an item out of a furnace.